meldingen_core/repositories.py

```diff
@@ -9,3 +9,11 @@ class BaseMeldingRepository(metaclass=ABCMeta):
     @abstractmethod
     def add(self, melding: Melding) -> None:
         ...
+
+    @abstractmethod
+    def list(self, *, limit: int | None = None, offset: int | None = None) -> list[Melding]:
```

Comment:
You can also use `typing.Optional[int]` here I think.

Reply:
Since python 3.10 you can also write `Optional` `as X | None`. See https://docs.python.org/3/library/typing.html#typing.Optional